File: NET/Kata.RockPaperScissor/Scissor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Kata.RockPaperScissor
{
    public class Scissor : Shape
    {
        public Scissor()
        {
            ShapeType = ShapeType.Scissor;
            Name = "Scissor";
            winsTo = new List<ShapeType>() { ShapeType.Paper, ShapeType.Lizard };
        }
    }
}

File: JavaScript/src/objects/scissor.js
var Shape = require('./shape');

var Scissor = Object.create(Shape);

Scissor.setup = function (ShapeType) {
	this.init({
		shapeType: ShapeType.scissor,
		winsTo: [ShapeType.paper, ShapeType.lizard]
	});
}

module.exports = Scissor;